tools/cli/datalab.py

```diff
@@ -170,6 +192,12 @@ def run():
                 dest='zone',
                 default=None,
                 help=_ZONE_HELP)
+        if command_config['require-email']:
+            subcommand_parser.add_argument(
+                '--email',
```

Comment:
A question:
- Is this only for git user -- if so git-user instead of email is likely much more descriptive/appropriate.

Thought:
- Do we actually need this? Won't the default always suffice, esp. with cloud source repository?